bot/interact_test.go

```diff
@@ -6,7 +6,17 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func requireHasError(t *testing.T, errorMessage string) {
+	require.NotEmpty(t, errorMessage)
+
+	if len([]rune(errorMessage)) > 150 {
```

Comment:
You can just do this, right?

```suggestion
	if len(errorMessage) > 150 {
```

Reply:
I just tried it out and it does work as expected. Thanks for the improvement!